content/en/events/upcoming-events/gsoc-2025.md

```diff
@@ -75,4 +75,306 @@ To participate in GSoC with Kubeflow, you __must__ meet the GSoC [eligibility re
 
 ## Project Ideas for 2025 GSoC
 
-Coming soon!
+### Project 1
+### Project Title: multi-tenancy/security, Kubeflow repository Migration, and CI/CD Enhancement
+
+#### Detailed Description:
+
+- Productionize the seaweedfs PoC as secure minio replacement https://github.com/kubeflow/manifests/tree/master/contrib/seaweedfs, since we are stuck with a 5 year old minio due to AGPL
+- add multi tenancy to seweadfs similar to the approach in feat(backend): isolate artifacts per namespace/profile/user using only one bucket pipelines#7725
+-Help to migrate Kubeflow/kubeflow components to kubeflow/dashboard and kubeflow/notebooks. This might even start before Gsoc
+- Renovate kubeflow/dashboard dependencies, addressing CVEs.
+- Implement architectural changes as needed.
+- Adapt and improve scripts and CI/CD in kubeflow/manifests, including matrix calls to test multiple Kubernetes versions simultaneously.
+
+#### Expected Outcomes:
+
+Successful migration of Kubeflow components.
+Hard multi-tenancy for the object storage
+Updated dependencies with resolved CVEs.
+Enhanced CI/CD pipeline supporting multiple Kubernetes versions.
+
+#### Skills Required/Preferred:
+
+GitHub and GitHub Actions
+Containerfile and Kubernetes knowledge
+Experience with Python, Go and JavaScript frameworks
+
+#### Mentors:
+
+@juliusvonkohout
+@thesuperzapper are you interested?
```

Comment:
@thesuperzapper are you interested in mentoring the migration part for Kubeflow/dashboard and kubeflow/notebooks?

Reply:
for now added @thesuperzapper name; we can correct it later if needs to.